services/src/main/java/org/apache/druid/cli/CliCoordinator.java

```diff
@@ -188,14 +189,9 @@ protected Set<NodeRole> getNodeRoles(Properties properties)
            : ImmutableSet.of(NodeRole.COORDINATOR);
   }
 
-  @Override
-  protected List<? extends Module> getModules()
+  protected static void validateCentralizedDatasourceSchemaConfig(Properties properties)
```

Comment:
Why not have this method in `ServerRunnable` instead?
If not needed, `CliX` should not need to refer to `CliCoordinator`.